Tapawingo_backend/Controllers/EditionsController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Tapawingo_backend.Services;
+using Tapawingo_backend.Dtos;
+
+namespace Tapawingo_backend.Controllers
+{
+    [ApiController]
+    public class EditionsController : Controller
+    {
+        private readonly EditionsService _editionsService;
+
+        public EditionsController(EditionsService editionsService)
+        {
+            _editionsService = editionsService;
+        }
+    
+            [HttpPost("organisations/{organisation_id}/events/{event_id}/editions")]
```

Comment:
Naar mijn weten wilden we niet meer deze langere urls. Volgensmij was het idee dat dit api/events/[id]/editions zou moeten worden